Mathlib/Algebra/BigOperators/Finprod.lean

```diff
@@ -411,7 +411,7 @@ theorem finprod_cond_ne (f : α → M) (a : α) [DecidableEq α] (hf : (mulSuppo
 
 @[to_additive]
 theorem finprod_mem_eq_prod_of_inter_mulSupport_eq (f : α → M) {s : Set α} {t : Finset α}
-    (h : s ∩ mulSupport f = t.toSet ∩ mulSupport f) : ∏ᶠ i ∈ s, f i = ∏ i ∈ t, f i :=
+    (h : s ∩ mulSupport f = ↑t ∩ mulSupport f) : ∏ᶠ i ∈ s, f i = ∏ i ∈ t, f i :=
```

Comment:
this might be a neater statement (no coe arrow necessary) if rephrased as 
```
(h : mulSupport f ∩ s = mulSupport f ∩ t)
```

Reply:
That's nice, but saving one character is probably not worth changing the statement and rename/deprecate the theorem ...